### PR TITLE
add links to changelog, finally

### PR DESCRIPTION
**Pull request by nicolaskruchten**
Can we please do this moving forward? It's a real pain right now to actually navigate from the changelog to the PRs :)

### CODE REVIEW

**Comment by archmoj** [comment]
It looks useful especially on the forks of `plotly.js`.

**Comment by archmoj** [comment]
On this page: https://github.com/plotly/plotly.js/blob/changelog_links/CHANGELOG.md
it appears clicking on the links does not move to the place where we want to go!

**Comment by archmoj** [comment]
> On this page: https://github.com/plotly/plotly.js/blob/changelog_links/CHANGELOG.md
> it appears clicking on the links does not move to the place where we want to go!

Basically we need to replace 
`https://github.com/plotly/plotly.js/pulls/#` with 
`https://github.com/plotly/plotly.js/pull/`

**Comment by nicolaskruchten** [comment]
fixed

**Comment by alexcjohnson** [comment]
That's half of it, still needs `pulls` -> `pull` 😁

**Comment by nicolaskruchten** [comment]
🤦 I swear I know how to do this!

**Comment by nicolaskruchten** [comment]
> It looks useful especially on the forks of plotly.js.

why? the #numbers don't work in this repo either :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,307 +13,307 @@ where X.Y.Z is the semver of most recent plotly.js release.
...
 ### Added
  - Removed usage of function constructors from `basic`, `cartesian`, `finance`, `geo`, and `mapbox`
-   partial bundles and added tests to ensure that they will not again do so in the future [#5359, #5383, #5387],
+   partial bundles and added tests to ensure that they will not again do so in the future [[#5359](https://github.com/plotly/plotly.js/pull/5359), [#5383](https://github.com/plotly/plotly.js/pull/5383), [#5387](https://github.com/plotly/plotly.js/pull/5387)],
    with thanks to [Equinor](https://www.equinor.com) for sponsoring the related development!
- - Add `strict` partial bundle [#5413, #5444], which includes
+ - Add `strict` partial bundle [[#5413](https://github.com/plotly/plotly.js/pull/5413), [#5444](https://github.com/plotly/plotly.js/pull/5444)], which includes
    the maximal subset of the library which does not rely on function constructors
- - Implement "fast" `zsmooth` option for `image` trace [#5354, #5386],
+ - Implement "fast" `zsmooth` option for `image` trace [[#5354](https://github.com/plotly/plotly.js/pull/5354), [#$1](https://github.com/plotly/plotly.js/pull/$1)],
```
Comment (alexcjohnson) [inline CHANGELOG.md:20]:
```suggestion
 - Implement "fast" `zsmooth` option for `image` trace [[#5354](https://github.com/plotly/plotly.js/pull/5354), [#5386](https://github.com/plotly/plotly.js/pull/5386)],
```

Comment (nicolaskruchten) [reply CHANGELOG.md]: fixed. too many undo/redo regex replacements

**Review by alexcjohnson — APPROVED** [review]
💃 😍

**Comment by alexcjohnson** [comment]
> > It looks useful especially on the forks of plotly.js.
> 
> why? the #numbers don't work in this repo either :)

I think @archmoj was just agreeing with what you're doing here :)

**Pull request merged**